src/main/resources/org/jenkinsci/plugins/environmentdashboard/DashboardBuilder/config.jelly

```diff
@@ -1,44 +1,38 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+	<f:entry title="Environment name" field="nameOfEnv">
+		<f:textbox />
+	</f:entry>
+	<f:entry title="Component name" field="componentName">
+		<f:textbox />
+	</f:entry>
+	<f:entry title="Build number" field="buildNumber">
+		<f:textbox />
+	</f:entry>
 
-        <f:entry title="Environment name" field="nameOfEnv">
-            <f:textbox />
-        </f:entry>
-        <f:entry title="Component name" field="componentName">
-            <f:textbox />
-        </f:entry>
-        <f:entry title="Build number" field="buildNumber">
-            <f:textbox />
-        </f:entry>
+	<f:entry title="Build job" field="buildJob">
+		<f:textbox />
+	</f:entry>
 
-        <f:entry title="Build job" field="buildJob">
-            <f:textbox />
-        </f:entry>
-
-        <f:entry title="Package Name" field="packageName">
-			<f:textbox />
-        </f:entry>
-
-        <f:nested>
-            <table>
-                <f:optionalBlock field="addColumns" title="Add Custom Columns" inline="true">
-                    <f:entry>
-                        <f:repeatable field="data" var="item" minimum="${1}" items="${descriptor.data}">
-                            <table>
-                                <f:entry field="columnName" title="${%Column Name}">
-                                    <f:textbox value="${item.columnName}" />
-                                </f:entry>
-                                <f:entry field="contents" title="${%Contents}">
-                                    <f:textbox value="${item.contents}"/>
-                                </f:entry>
-                                <f:entry>
-                                    <div align="right"> <f:repeatableDeleteButton /> </div> 
-                                </f:entry>
-                            </table>
-                        </f:repeatable>
-                    </f:entry>
-                </f:optionalBlock>
-            </table>
-        </f:nested>
+	<f:entry title="Package Name" field="packageName">
+		<f:textbox />
+	</f:entry>
 
+	<f:nested>
+		<f:optionalBlock field="addColumns" title="Add Custom Columns" inline="true">
+			<f:entry>
```

Comment:
there's migration guides here: https://www.jenkins.io/doc/developer/views/table-to-div-migration/

basically:

table => div
tr => div class="tr"
td => div

and keep the rest of the markup the same (not strictly needed if you're dropping support for old core, but normally easier to stay compatible if you don't change the markup too)

you'll want to use a local taglib so that the right tags will be used where required

here's a full example doing it for table:
https://github.com/jenkinsci/git-plugin/pull/973/files

duplicate the tag pretty much for tr and td, we normally call it rowWrapper and entryWrapper

also you're IDE has changed all  the tabs to spaces would be good if that could be avoided

here's another example where they are extracted into a standalone taglib:
https://github.com/jenkinsci/postbuild-task-plugin/pull/6

Reply:
Ah cool I will take a look, I was just experimenting with getting our work install running properly, I will make these changes in the morning